### DISCUSSION TITLE
How to exclude folder on windows version?

### DISCUSSION THREAD

**Discussion by Slicedeye** [theophanemayaud/video-simili-duplicate-cleaner#135] [Q&A]
Hi, I have thousands of folder with hundreds of thousands of videos and i would like to exclude some of the folders instead of just selecting a few.
How can I accomplish this?

**Comment by theophanemayaud** [comment]
Hey and thanks for reaching out! You can't exclude folder to be scanned, but what you can do is once you have scanned and gotten some results, you can exclude folders in the comparison window, from being "skipped" in the comparisons. Those exclusion options are in the third tab of the comparison window
